gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -216,6 +216,12 @@
   public static final String QUEUED_TASK_TIME_MAX_AGE = "taskexecutor.queued_task_time.history.max_age";
   public static final long DEFAULT_QUEUED_TASK_TIME_MAX_AGE = TimeUnit.HOURS.toMillis(1);
 
+  /**
+   * Optional property to specify for Mysql ingestion whether insertions should do replace logic for records
+   * with the same PRIMARY KEY or unique identifier.
+   */
+  public static final String REPLACE_FOR_MYSQL_INGESTION = "replace.allowed.for.mysql.ingestion";
```

Comment:
also, be sure to note in the documentation that replacement granularity is 'record-level', meaning that if there are duplicates in the input (by primary key) they'd even replace one another.  hence which record ultimately gets loaded is undefined (since they may be in separate WUs, which are not processed in a specific order).